InformationScripting/src/graph/Property.cpp

```diff
@@ -33,5 +33,10 @@ boost::python::object pythonObject(const Property& p)
 	return p.data_->pythonObject();
 }
 
+bool Property::operator==(const Property& other) const
```

Comment:
Well InformationNode is a PropertMap, but yeah as you say with this you can compare all properties. For the definition of equality of nodes we will have to discuss together with the hashing.